resources/views/Staff/uploader/index.blade.php

```diff
@@ -0,0 +1,52 @@
+@extends('layout.default')
+
+@section('title')
+    <title>
+        {{ __('torrent.uploader') }} {{ __('common.stats') }} - {{ config('other.title') }}
+    </title>
+@endsection
+
+@section('breadcrumbs')
+    <li class="breadcrumbV2">
+        <a href="{{ route('staff.dashboard.index') }}" class="breadcrumb__link">
+            {{ __('staff.staff-dashboard') }}
+        </a>
+    </li>
+    <li class="breadcrumb--active">{{ __('torrent.uploader') }} {{ __('common.stats') }}</li>
+@endsection
+
+@section('page', 'page__uploader-index')
+
+@section('main')
+    <section class="panelV2">
+        <header class="panel__header">
+            <h2 class="panel__heading">{{ __('torrent.uploader') }} {{ __('common.stats') }}</h2>
+        </header>
+        <div class="data-table-wrapper">
+            <table class="data-table">
+                <thead>
+                    <tr>
+                        <th>{{ __('common.name') }}</th>
+                        <th>{{ __('user.total-uploads') }}</th>
+                        <th>{{ __('user.uploads') }} {{ __('stat.last60days') }}</th>
+                        <th width="15%">{{ __('common.action') }}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    @foreach ($uploaders as $uploader)
+                        <tr>
+                            <td><x-user_tag :anon="false" :user="$uploader" /></td>
+                            <td>
+                                {{ $uploader->total_uploads }}
+                            </td>
+                            <td>
+                                {{ $uploader->recent_uploads }}
+                            </td>
```

Comment:
I don't agree with these stats. Better quality uploaders will have way fewer uploads but most will be personal releases (Please add another column for personal release count). It would also good to have a breakdown of how many of each `type` of torrent have been uploaded per uploader within the time period.

Reply:
Added the personal releases, too.

I did not yet include the uploads per type, yet. I like the idea, but I'd add this as it is now and improve with more information within a new PR.